MekHQ/src/mekhq/gui/dialog/MedicalViewDialog.java

```diff
@@ -116,16 +116,14 @@
     private transient Font handwritingFont;
     private transient Color labelColor;
     private transient ImageIcon healImageIcon;
-    
-    private boolean gmMode;
 
     public MedicalViewDialog(Window parent, Campaign c, Person p, IconPackage ip) {
         super();
         this.campaign = Objects.requireNonNull(c);
         this.person = Objects.requireNonNull(p);
         //this.iconPackage = Objects.requireNonNull(ip);
```

Comment:
Do we need this anymore?

Reply:
I don't know. Outside the scope of this PR.